contrib/devtools/github-merge.sh

```diff
@@ -83,12 +83,13 @@ function cleanup() {
 
 # Create unsigned merge commit.
 (
-  echo "Merge pull request #$PULL"
+  PRTITLE=`curl -s https://api.github.com/repos/$REPO/pulls/$PULL | grep -e '  "title": ".*",'| awk -F'"' '{print $4}'`
```

Comment:
This needs to have the organisation in the URL. i.e api.github.com/repos/bitcoin/bitcoin/pulls/5623 As is, it just returns "Not Found" messages.